doc/config.rst

```diff
@@ -333,9 +333,10 @@ Project information
    .. versionadded:: 0.5
 
    .. versionchanged:: 1.4
-      The default is now ``'python3'``, since it is mostly a superset of
-      ``'python'``.  If you prefer Python 2 only highlighting, you can set
-      it back to ``'python'``.
+      The default is now ``'default'``. It is similar to ``'python3'``;
+      it is mostly a superset of ``'python'``. but it fallbacks to
+      ``'none'`` if failed.  If you prefer Python 2 only highlighting,
```

Comment:
I think this should state that this falls back to "none" without warning, while the others do warn.